cmd/storage-rpc-client.go

```diff
@@ -318,6 +325,5 @@ func newStorageRPC(endpoint Endpoint) *StorageRPCClient {
 	logger.FatalIf(err, "Unable to parse storage RPC Host", context.Background())
 	rpcClient, err := NewStorageRPCClient(host, endpoint.Path)
 	logger.FatalIf(err, "Unable to initialize storage RPC client", context.Background())
```

Comment:
If you are changing this then all assumptions of initStorageDisks has to be changed. Since after healing, connecting to servers after disconnects all have change in behavior.

I would suggest we avoid that and instead print always when there is a problem not after 5 retries I don't see any value in not printing until 5 retries.

Reply:
> If you are changing this then all assumptions of initStorageDisks has to be changed. Since after healing, connecting to servers after disconnects all have change in behavior.
> 
> I would suggest we avoid that and instead print always when there is a problem not after 5 retries I don't see any value in not printing until 5 retries.

Using retry is useful to avoid printing "connection refused" error too early, which is a normal error if we run the distributed mode, not all nodes will be ready at the same time.

At the same time, I don't want to remove printing the "connection refused" error since it can be useful sometimes, like when a user makes a typo in the IP address or a port of a specific endpoint.

I guess I can take a look of the behavior change problem and fix it.

Reply:
It is generally okay to print something informative like connect refused, the problem I have with 5 or more retries is the changes like Connect() function. They make everything super complex. 

Knowing such a detail outside of RPC client makes it hard, now one has to remember to do it after newStorageRPC it can get pretty complex.

Reply:
Yes, I agree.

What about adding LastError() instead of Connect() ? LastError() will return plain error instead of the generic storage errors, like: errDiskNotFound

Reply:
We can do that..